spec.bs

```diff
@@ -282,6 +297,16 @@ navigator.credentials.create({ publicKey })
 
 </div> <!-- non-normative -->
 
+**TODO**: Sugest we provide clarifying text/expand around the following 
+* Does the extension imply that this credential my ONLY be used for Payment, or does this mean the WebAuthn credential could be used for Login or Payment, 
```

Comment:
Seems reasonable to clarify briefly. It can be used for both login and payment, if the RP wants to. They can also trivially not use it for login; if doing that we recommend they utilize a subdomain (e.g. `payment.bank.com`) to avoid polluting their credential space with credentials they wouldn't accept for login.